<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reservations</class>
 <widget class="QWidget" name="Reservations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
    border: none;
    background-color: transparent;
    background: none;
    padding: 0;
    margin: 0;
}

#centralwidget {
    background-color: #FAFAFA; /* Light gray for the main background */
}

#LeftMenuContainer {
    background-color: #F5F5F5; /* Slightly darker light gray for the sidebar */
}

QPushButton {
    background-color: #E8F0FE;  /* Light blue for buttons */
    color: #333333;            /* Dark gray text for readability */
    border: none;
    border-radius: 8px;        /* Rounded corners */
    padding: 12px 20px;        /* Balanced padding */
    font-size: 15px;           /* Medium font size for modern feel */
    font-weight: bold;         /* Emphasized text */
}

QPushButton:hover {
    background-color: #D7E6FD; /* Softer blue when hovered */
}

QPushButton:pressed {
    background-color: #C6D9FB; /* Slightly darker blue when pressed */
    color: #1A1A1A;            /* Darker text for contrast */
}

QPushButton:disabled {
    background-color: #F0F0F0; /* Muted gray for disabled buttons */
    color: #A8A8A8;            /* Muted text color */
}

QLineEdit {
    border: 2px solid green; /* Border color */
    border-radius: 5px; /* Rounded corners */
    padding: 5px; /* Internal padding for text */
    background-color: white; /* Background color */
    color: black; /* Text color */
    font: bold 12px; /* Font style and size */
}

QLineEdit:focus {
    border: 2px solid #32CD32; /* Bright green border on focus */
    background-color: white; 
}

QTextEdit:hover {
    border: 2px solid darkgreen; /* Border color on hover */
    background-color: #E6FFE6; /* Light hover effect */
}
QComboBox {
    border-radius: 5px;
    padding: 7px;
    color: white;
    text-align: center; /* Center-align the text */
}

QComboBox:hover {
    border-color: darkgreen;
}

QComboBox::drop-down {
    width: 20px; /* Adjust width of the dropdown button */
    border: none;
}

QComboBox QAbstractItemView {
    color: black;
 
    selection-color: black;
    padding: 5px;
    text-align: center;
    outline: none; /* Remove focus border for cleaner look */
}

QWidget {
       background-color: #fefefe;
       font-family: 'Segoe UI';
       font-size: 14px;
}

QLabel {
       font-weight: 600;
       color: #222;
 }

QLineEdit, QSpinBox, QComboBox, QDateEdit, QTimeEdit {
       padding: 6px;
       border: 1px solid #ccc;
       border-radius: 6px;
       background-color: white;
}

QPushButton {
    border-radius: 10px; /* Rounded corners */
    color: white; /* Text color */
    background-color: green; /* Background color */
    font: bold 13px; /* Font style and size */
    border: 2px solid green; /* Border color */
}

QPushButton:hover {
    background-color: #32CD32; /* Lighter green on hover */
    border-color: darkgreen; /* Dark green border on hover */
}

QPushButton:pressed {
    background-color: darkgreen; /* Darker green when pressed */
    border-color: black; /* Border changes on press */
}
  
QTableView {
    border-radius: 10px; /* Rounded edges for the entire table */
    padding: 10px; /* Adds spacing around the table */
    border: 1px solid #B0C4DE; /* Thin border to accentuate table outline */
    gridline-color: black; /* Adds black lines between columns and rows */
}

QTableView::item {
    background-color: #FAFAFA; /* Gentle off-white background for cells */
    border: 1px solid black; /* Black lines between each cell */
    border-radius: 5px; /* Slight rounding for individual cell edges */
    padding: 5px; /* Inner spacing for cells */
    color: black; /* Standard text color */
}

QTableView::item:selected {
    background-color: #D7E9F7; /* Highlighted cell with soft blue shade */
    color: black; /* Text color remains consistent */
    border: 1px solid black; /* Black border for selected cells */
    border-radius: 5px;
}

QHeaderView::section {
    background-color: #B0C4DE; /* Soft blue-gray shade for table headers */
    padding: 10px; /* Adds spacing within header sections */
    font-weight: bold; /* Bold text to differentiate headers */
    color: black; /* Black text color for the headers */
    border: 1px solid black; /* Black lines between header columns */
  
}

QTableView::corner {
    background-color: #B0C4DE; /* Matches header color for corner sections */
    border-radius: 10px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="font">
      <font>
       <family>Leelawadee UI</family>
       <pointsize>22</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 22pt &quot;Leelawadee UI&quot;;
background-color: #E8F0FE;
border-radius: 10px;
color: rgb(0, 0, 0);
padding: 5px;</string>
     </property>
     <property name="text">
      <string>  Reservation</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="datetimeLayout">
     <item>
      <widget class="QLabel" name="label_date">
       <property name="text">
        <string>Select Date:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_date">
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
       <property name="displayFormat">
        <string>dd/mm/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="timeEdit_time">
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_customerDetails">
     <property name="styleSheet">
      <string notr="true">font: 600 16pt &quot;Segoe UI&quot;;
background-color: #E8F0FE;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string> Customer Details</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_guests1">
       <property name="text">
        <string>Guests:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="reservation_contact">
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_contact">
       <property name="text">
        <string>Contact:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_guests">
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="reservation_name">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>-1</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget_tables">
     <property name="styleSheet">
      <string notr="true">background-color: #E8F0FE;
</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>6</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>32</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Table No.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Guests</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Contact</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomLayout">
     <item alignment="Qt::AlignmentFlag::AlignRight">
      <widget class="QLabel" name="label_selectTable">
       <property name="text">
        <string>Select Table:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_selectTable">
       <property name="styleSheet">
        <string notr="true">QComboBox {
    border-radius: 5px;
    padding: 7px;
    color: black;
    background-color: white;
    text-align: center; /* Center-align the text */
}

QComboBox:hover {
    border-color: gray;
    background-color: #F5F5F5; /* Lighter white shade for hover effect */
}

QComboBox::drop-down {
    width: 20px; /* Adjust width of the dropdown button */
    border: none;
}

QComboBox QAbstractItemView {
    background-color: white;
    color: black;
    selection-background-color: #E8E8E8; /* Highlight selected item in a soft white shade */
    selection-color: black;
    padding: 5px;
    text-align: center;
    outline: none; /* Remove focus border for cleaner look */
}
</string>
       </property>
       <item>
        <property name="text">
         <string>Table 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Table 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Table 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Table 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Table 5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Table 6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_check">
       <property name="text">
        <string>Check Availability</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_reserve">
       <property name="text">
        <string>Reserve Table</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
